plugins/vpp/l3plugin/vppcalls/vpp1908/route_dump.go

```diff
@@ -172,18 +137,12 @@ func (h *RouteHandler) dumpRouteIPDetails(tableID uint32, tableName []byte, addr
 		}
 	} else {
 		// Return route without path fields, but this is not a valid configuration
-		h.log.Warnf("Route with destination IP %s (VRF %d) has no path specified", dstIP, tableID)
```

Comment:
Seems like this warning was imporant, or is the case for returning route with no paths expected now?

Reply:
No, it's still the invalid case, I'll keep it there.